packages/truffle/test/sources/init/contracts/Migrations.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+contract Migrations {
+  address public owner = msg.sender;
```

Comment:
we can speed up tests if we remove migrations. Though we should have 1 integration test to cover that path.